### DISCUSSION TITLE
If the .cache folder was cleared do installed packages from AUR still update with paru -Sua?

### DISCUSSION THREAD

**Discussion by JuniorSum1** [Morganamilo/paru#1198] [Q&A]
I cleared the .cache directory and I'm concerned that AUR packages will not longer update because the ~/.cache/paru directory is deleted. I reinstalled all the pacman -Qm packages with paru to regenerate it. Should I stop clearing the .cache directory?

**Comment by Morganamilo** [comment]
cache is not important by design. Only downside to deleting it is you'll have to download that stuff again on package update.
